machines/bleShare/request/requestMachine.ts

```diff
@@ -241,11 +252,21 @@ export const requestMachine =
             },
             CONNECTED: {
               target: 'waitingForVc',
-              actions: ['setSenderInfo', 'setReceiverInfo'],
+              actions: [
+                'setSenderInfo',
+                'setReceiverInfo',
+                'sendVcSharingStartEvent',
```

Comment:
we should rename this to sendVcReceiveStartEvent.

Reply:
We have not Vc receive till now. It's just in connect state. Named it VCShare as the flow is VC share